### PR TITLE
Factions for animals

### PR DESCRIPTION
**Pull request by Nimfar11**
## About the PR

Added a faction to some animals so they too would be targeted and not use their neutral status to attack with impunity. Added a new faction to be the target of an attack, but not to attack anyone. 
Made some factions enemies to each other.

Solves the problem [#15988](https://github.com/space-wizards/space-station-14/issues/15988)

- [X] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

:cl: Nimfar11

- add: Adds a neutral faction Delicious, to Monkeys. Now they are not untouchable. They are a delicacy for the antagas.
- add: Adds a faction PetsNT for the cat Exception, Remilia and Cerberus.

### CODE REVIEW

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
- Changelog needs updating
- Alphabetical please
- Cerberus and Remi should not be passive by default considering they're based around being able to do combat
- Victim isn't a good name considering it should be something everything is hostile to

File: Resources/Prototypes/Entities/Mobs/Player/familiars.yml [hunk 1]
```diff
@@ -53,7 +56,7 @@
   - type: MobMover
   - type: Faction
     factions:
-    - SimpleNeutral
+    - PetsNT
```
Comment (metalgearsloth) [inline Resources/Prototypes/Entities/Mobs/Player/familiars.yml:59]: Cerberus isn't an NT pet, it's an antag pet.

**Review by metalgearsloth — CHANGES_REQUESTED** [review]
- "Delicious" isn't a good name, just call it "Passive"
- Still needs to be alphabetical

File: Resources/Prototypes/Entities/Mobs/Player/familiars.yml (hunk 1 above)
Comment (Nimfar11) [reply Resources/Prototypes/Entities/Mobs/Player/familiars.yml:59]: You mean give him back a neutral faction? The AI should not attack players on his own.

File: Resources/Prototypes/ai_factions.yml [hunk 2]
```diff
@@ -29,6 +33,8 @@
   hostile:
   - NanoTrasen
   - Syndicate
+  - PetsNT
+  - Passive
```
Comment (metalgearsloth) [inline Resources/Prototypes/ai_factions.yml:37]:
```suggestion
  - Passive
  - PetsNT
```

File: Resources/Prototypes/ai_factions.yml [hunk 3]
```diff
@@ -39,9 +45,12 @@
...
   - Xeno
+  - PetsNT
 
 - type: faction
   id: Xeno
   hostile:
   - NanoTrasen
   - Syndicate
+  - PetsNT
+  - Passive
```
Comment (metalgearsloth) [inline Resources/Prototypes/ai_factions.yml:56]:
```suggestion
  - Passive
  - PetsNT
```

**Pull request merged**